DatePicker/tests/DatePicker-specs.js

```diff
@@ -1,189 +1,146 @@
 		const label = 'custom year aria-label';
-		const subject = mount(
-			<DatePicker value={new Date(2000, 0, 1)} yearAriaLabel={label} />
-		);
-
-		const yearPicker = subject.find(`DateComponentRangePicker.${css.year}`);
-
-		const expected = label;
-		const actual = yearPicker.prop('aria-label');
-
-		expect(actual).toBe(expected);
-	});
-
-	test('should set "monthAriaLabel" to month picker', () => {
-		const label = 'custom month label';
-		const subject = mount(
-			<DatePicker monthAriaLabel={label} value={new Date(2000, 0, 1)} />
-		);
-
-		const monthPicker = subject.find(`DateComponentRangePicker.${css.month}`);
-
-		const expected = label;
-		const actual = monthPicker.prop('aria-label');
-
-		expect(actual).toBe(expected);
-	});
-
-	test('should set "yearAriaLabel" to year picker', () => {
-		const label = 'custom year label';
-		const subject = mount(
+		render(
 			<DatePicker value={new Date(2000, 0, 1)} yearAriaLabel={label} />
 		);
+		const yearPicker = screen.getByLabelText(label);
 
-		const yearPicker = subject.find(`DateComponentRangePicker.${css.year}`);
+		const expected = 'year';
 
-		const expected = label;
-		const actual = yearPicker.prop('aria-label');
-
-		expect(actual).toBe(expected);
+		expect(yearPicker).toBeInTheDocument();
+		expect(yearPicker).toHaveClass(expected);
 	});
 
 	test('should set "data-webos-voice-disabled" to day picker when voice control is disabled', () => {
-		const subject = mount(
+		render(
 			<DatePicker value={new Date(2000, 0, 1)} data-webos-voice-disabled />
 		);
+		const dayPicker = screen.getByLabelText('1 day change a value with up down button');
 
-		const dayPicker = subject.find(`DateComponentRangePicker.${css.day}`);
-
-		const expected = true;
-		const actual = dayPicker.prop('data-webos-voice-disabled');
+		const expected = 'data-webos-voice-disabled';
 
-		expect(actual).toBe(expected);
+		expect(dayPicker).toHaveAttribute(expected);
 	});
 
 	test('should set "data-webos-voice-disabled" to month picker when voice control is disabled', () => {
-		const subject = mount(
+		render(
 			<DatePicker value={new Date(2000, 0, 1)} data-webos-voice-disabled />
 		);
+		const monthPicker = screen.getByLabelText('1 month change a value with up down button');
 
-		const monthPicker = subject.find(`DateComponentRangePicker.${css.month}`);
+		const expected = 'data-webos-voice-disabled';
 
-		const expected = true;
-		const actual = monthPicker.prop('data-webos-voice-disabled');
-
-		expect(actual).toBe(expected);
+		expect(monthPicker).toHaveAttribute(expected);
 	});
 
 	test('should set "data-webos-voice-disabled" to year picker when voice control is disabled', () => {
-		const subject = mount(
+		render(
 			<DatePicker value={new Date(2000, 0, 1)} data-webos-voice-disabled />
 		);
+		const yearPicker = screen.getByLabelText('1900 year change a value with up down button');
+		// The year is 1900 because it does not change based on props, this needs to be fixed in datePicker
```

Comment:
Yeah, I know but I assumed this has been a known issue since the test case above "test('should accept a JavaScript Date for its value prop'" uses locale for checking the correct year. I guess it needs locale information to show the correct date. We don't use DatePicker without locale so the issue won't exist in the real scenario. Does it make sense to you?